src/components/ResolutionModal/ResolutionStep.js

```diff
@@ -45,45 +46,59 @@ class ResolutionStep extends Component {
         const { resolutions, selected } = this.state;
 
         if (!resolutions) {
-            return null; // TODO loading
+            return (
+                <Stack gutter='sm'>
+                    <StackItem><h1>{ this.issue.description }</h1></StackItem>
+                    <StackItem><h2>Would you like to:</h2></StackItem>
+                    <StackItem>
+                        <Skeleton/>
+                    </StackItem>
+                    <StackItem>
+                        <Skeleton/>
+                    </StackItem>
+                    <StackItem>
+                        <Skeleton/>
+                    </StackItem>
+                </Stack>
+            );
+        } else {
```

Comment:
<nitpick>Since the branch above uses return it is not necessary to use else here. Using else leads to more nesting which can be harder to read. https://eslint.org/docs/rules/no-else-return

Reply:
OK! Cleaning that up now